contrib/ansible/docker-compose.yml.j2

```diff
@@ -0,0 +1,38 @@
+version: "3.3"
+services:
+    db:
```

Comment:
(ever more minor) You use 2 space indentation in Ansible playbook, can you please use 2 space indention in this docker file? I think it's more than ok for YAMLs. :) But you can keep this way if you like, just a suggestion.

Reply:
Ok. Will do!